Makefile

```diff
@@ -1,53 +0,0 @@
-.PHONY: help prepare-dev init test run
-
-# Python stuff
-VENV_NAME?=.venv
-PYTHON3 := $(shell command -v python3 2> /dev/null)
-VENV_ACTIVATE=$(VENV_NAME)/bin/activate
-PYTHON=$(VENV_NAME)/bin/python3
```

Comment:
Should we clean up Python stuff in .gitignore? https://github.com/nimblehq/flutter-templates/blob/00b095d47ef275ec00d34faecd78efa4436fcb1b/.gitignore#L4-L6

Reply:
@luongvo nice catch. Updated in 72792b8